recommenders/datasets/movielens.py

```diff
@@ -196,6 +194,17 @@ def load_pandas_df(
         warnings.warn(WARNING_MOVIE_LENS_HEADER)
         header = header[:4]
 
+    if size in MOCK_DATA_FORMAT:
+        # function-wide import to isolate extra dependencies from the mock schema will use
+        from recommenders.datasets.mock.movielens import MockMovielensSchema
```

Comment:
This import statement is probably better at the beginning of the file according to PEP guidelines.

Reply:
Removed